forest/src/cli/chain_cmd.rs

```diff
@@ -82,18 +85,32 @@ impl ChainCommands {
 
                 let epoch = tipset.unwrap_or(chain_head.epoch());
 
+                let now = OffsetDateTime::now_utc();
+
+                let month = now.month() as u8;
+                let month_string = if month < 10 {
+                    format!("0{}", month)
+                } else {
+                    month.to_string()
+                };
+                let year = now.year();
+                let day = now.day();
+                let chain_name = cfg.chain.name.clone();
+
+                let mut vars = HashMap::new();
```

Comment:
1. Do we always need to build this map and pay for it? Or can we do it *only* if we reach `Some(path)` in a `match` later on?
2. That's a lot of cloning, would be nice if we could reduce it. It's not a performance-critical path though, so it's not a big deal.

Reply:
Done both